dask/utils.py

```diff
@@ -85,13 +140,9 @@ def ndeepmap(n, func, seq):
         return func(seq)
 
 
+@_deprecated(version="2021.06.1", use_instead="contextlib.supress")
```

Comment:
In the original message you had:
```suggestion
@_deprecated(version="2021.06.1", use_instead="contextlib.supress from the standard library")
```

I mildly prefer that version :)